app/helpers/application_helper.rb

```diff
@@ -32,14 +32,16 @@ def accept_friendship(friendship)
     show_user = User.find_by(id: params[:id])
     (return unless current_user == show_user)
 
-    link_to('Accept?', user_friendship_path(user_id: friendship.id), method: :put, class: 'profile-link')
+    # link_to('Accept?', user_friendship_path(user_id: friendship.id), method: :put, class: 'profile-link')
+    link_to('Accept?', user_friendship_path(user_id: friendship.user.id), method: :put, class: 'profile-link')
   end
 
   def reject_friendship(friendship)
     show_user = User.find_by(id: params[:id])
     (return unless current_user == show_user)
 
-    link_to('Reject?', user_friendship_path(user_id: friendship.id), method: :delete, class: 'profile-link')
+    # link_to('Reject?', user_friendship_path(user_id: friendship.id), method: :delete, class: 'profile-link')
```

Comment:
Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.